src/hearings/containers/request-hearing/hearing-final-confirmation/hearing-final-confirmation.component.ts

```diff
@@ -18,6 +19,10 @@ export class HearingFinalConfirmationComponent implements OnInit {
   constructor(protected readonly hearingStore: Store<fromHearingStore.State>) {
   }
 
+  public get hearingConfirmationSourceEnum() {
+    return HearingConfirmationSource;
+  }
+
```

Comment:
this probably can be removed and not necessary

Reply:
I have removed this code as it is no more required.